wemake_python_styleguide/cli/commands/explain/module_loader.py

```diff
@@ -0,0 +1,43 @@
+import importlib
+from collections.abc import Collection
+from pathlib import Path
+from types import ModuleType
+from typing import Final
+
+_VIOLATION_MODULE_BASE: Final = 'wemake_python_styleguide.violations'
+
+
+def get_violation_submodules() -> Collection[ModuleType]:
+    """Get all possible violation submodules."""
+    return _safely_get_all_submodules(_VIOLATION_MODULE_BASE)
```

Comment:
Do we really need `_safely_get_all_submodules`? It is used in a single place, but all this function does is calling `_safely_get_all_submodules`

So, let's just remove `_safely_get_all_submodules` and move all code to `get_violation_submodules`